### ISSUE TITLE
don't use multiple yaml documents

### ISSUE THREAD

**Issue by soenkehahn** [Originate/check-protocols#88]
Currently, multiple protocols can be specified by putting them into multiple documents in the same yaml file:

``` yaml
protocol:
  - foo
  - bar
---
protocol:
  - bar
  - baz
```

But for newcomers this seems to be confusing and is maybe generally more an obscure yaml feature? So maybe it'd be better to just switch to:

``` yaml
- protocol:
  - foo
  - bar
- protocol:
  - bar
  - baz
```

The question is whether we then want to disallow specifying a single protocol as a top-level array of steps like this:

``` yaml
- foo
- bar
```

Because that'd be hard to parse and confusing.

cc: @xasopheno

**Comment by matthandlersux** [comment]
i don't think the multi-doc `---` is hard to pick up, especially if it's in examples and the readme, however, i agree with your comment on slack that having multiple documents makes it less straightforward to have "top level" parts of a protocol file that apply to all tests within the file (like how #89 and #72 might work). i also don't care much about the shorthand version yet... maybe when i try it out on a big project it will become more desired.

so my vote would be for the example in the middle

**Comment by xasopheno** [comment]
I also like the one in the middle.
